package.json

```
@@ -13,8 +13,8 @@
     "build:crypto-dependencies": "lerna run --scope '@aws-sdk/types' --scope '@aws-sdk/util-utf8-browser' --scope '@aws-sdk/util-locate-window' --scope '@aws-sdk/hash-node' --include-dependencies pretest",
     "build:protocols": "yarn build:crypto-dependencies && lerna run --scope '@aws-sdk/aws-*' --include-dependencies pretest",
     "build:smithy-client": "yarn build:crypto-dependencies && lerna run --scope '@aws-sdk/client-rds-data' --include-dependencies pretest",
-    "build:all": "yarn build:crypto-dependencies && lerna run pretest",
-    "pretest:all": "yarn build:all",
+    "build:all": "yarn build:crypto-dependencies && lerna run build:cjs && lerna run build:es",
```

Add `build` command in all `packages/*` which runs `build:cjs` & `build:es`
```suggestion
    "build:all": "yarn build:crypto-dependencies && lerna run build",
```

It would be similar to that done in clients (although clients needs to be corrected to introduce `build:cjs` and can be done in a different PR.

https://github.com/aws/aws-sdk-js-v3/blob/b276ec19f7251f897240bc78b2d6fdfe7abaa618/clients/client-accessanalyzer/package.json#L17